.github/workflows/bump-preview.yml

```diff
@@ -0,0 +1,25 @@
+name: Preview API documentation
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'oas_docs/kibana.serverless.yaml'
+
+jobs:
+  api-preview:
+    if: contains(github.event.pull_request.labels.*.name, 'ci:api-preview')
+    name: Generate preview on Bump.sh
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Create Preview
+        uses: bump-sh/github-action@v1
+        with:
+          doc: serverless
+          token: ${{secrets.BUMP_TOKEN}}
```

Comment:
@lcawl I believe this would fail when ran from forks. Since it won't have access to the secret.

Reply:
Since this PR doesn't seem to solve the problem, I've changed it to remove the existing problematic workflow instead.